Add docker sec

huynaism opened the pull request:
- Fixing Hadolint issues
- Add trivyignore

filippos47 commented on .github/workflows/publish.yml:
```diff
@@ -18,8 +19,13 @@ jobs:
       run-lint: true
 
   docker_pipeline:
```
```suggestion
  docker_pipeline:
    needs: ["lint_test"]  
```

filippos47 reviewed (commented):
Let's:
- Add go-ethereum to trivyignore
- Add a CHANGELOG to fix the broken pipeline

All the rest should be now fixed.

huynaism commented:
> Let's:
> 
> * Add go-ethereum to trivyignore
> * Add a CHANGELOG to fix the broken pipeline
> 
> All the rest should be now fixed.

ok all resolved, added Geth ignore in  116426df423a188098b0e2df7ed544797186f7c2

RafilxTenfen commented:
@filippos47 @hiepmai-babylonchain can this be closed or merged?
